project.yaml

```diff
@@ -14,8 +16,7 @@ schema:
   file: ./schema.graphql
 network:
   chainId: "0x97da7ede98d7bad4e36b4d734b6055425a3be036da2a332ea5a7037656427a21"
-  endpoint: [wss://nodle-parachain.api.onfinality.io/public-ws]
-  dictionary: https://api.subquery.network/sq/subquery/nodle-parachain-dictionary
```

Comment:
No use for the dictionary any more?

Reply:
Subquery detects the dictionary automatically by taking the chain Id, we can added it tho (like forcing it)